src/javascript/model/model.js

```diff
@@ -0,0 +1,73 @@
+import { ERROR_MESSAGE } from '../utils/constants.js';
+export default class Model {
+  constructor(storages) {
+    this._storages = storages;
+    this._todos = {};
+    for (let userName in storages) {
+      this._todos[userName] = storages[userName].todos;
+    }
+  }
+
+  async create(value, userName) {
+    if (value.length === 0) {
```

Comment:
```suggestion
    if (!value.length) {
```

매직 넘버를 넣는것 보다는 Javascript의 `falsy` 를 이용해서 더 간결하고 깔끔하게 작성할 수 있을거 같아요!

- [거짓같은 값 / MDN](https://developer.mozilla.org/ko/docs/Glossary/Falsy)

Reply:
알고있는건데도 습관 못고치고 계속 이렇게 사용하네😢😢 짚어줘서 떙큐~~